app/pb_kits/playbook/_playbook.scss

```
@@ -81,4 +82,4 @@
 @import 'pb_user_badge/user_badge';
 @import 'pb_time_stacked/time_stacked';
 @import 'pb_weekday_stacked/weekday_stacked';
-@import './utilities/spacing';
+@import './utilities/spacing';
```

```suggestion
@import './utilities/spacing';

```

We should not remove the extra line.